docs/intro.rst

```diff
@@ -54,6 +54,4 @@ Some of the main differences include:
 * It is possible to extend the schemata of existing Archetypes types with the archetypes.schemaextender product, although this adds some performance overhead and relies on a somewhat awkward programming technique. Dexterity types were built to be extensible from the beginning, and it is possible to declaratively turn on or off aspects of a type (such as Dublin Core metadata, locking support, ratings, tagging, etc) with re-usable "behaviors".
 * Dexterity is built from the ground up to support through-the-web type creation. There are products that achieve the same thing with Archetypes types, but they have to work around a number of limitations in the design of Archetypes that make them somewhat brittle or slow. Dexterity also allows types to be developed jointly through-the-web and on the filesystem. For example, a schema can be written in Python and then extended through the web.
 
-All that said, Archetypes is still an older and more mature framework. Dexterity is a viable alternative to Archetypes in many projects, but if you are happy with Archetypes, you should not immediately rush to rewrite your types to use Dexterity.
-
-There are also some things that Dexterity does not yet support, or, more commonly, services that Plone ships with that currently assume all content objects are built using Archetypes. The current list of "gaps" can be found in the Dexterity issue tracker. You should take a look at this before deciding whether Dexterity will work for you. If in doubt, don't hesitate to write to the Dexterity mailing list and ask for advice.
+As of version 5 of Plone, Dexterity is the preferred way of creating content types.  Additionally, Archetypes was removed from Plone core in 5.2.  It can still be added to Plone to support Archetypes-based add-ons, but it will not function when running Plone using Python 3.   If you are considering upgrading to Plone 5 and Python 3, you are reading the right documentation.
```

Comment:
Content is fine, but please follow the doc style for all new edits; here: One sentence per line. 
https://docs.plone.org/develop/styleguide/documentation.html

Reply:
thanks!

Reply:
https://docs.plone.org/about/contributing/documentation_styleguide.html is correct link to documentation styleguide.